templates/internet-of-things/index.html

```diff
@@ -25,297 +27,232 @@ <h1>Ubuntu for the Internet&nbsp;of&nbsp;Things</h1>
+                height="150",
                 hi_def=True,
-              ) | safe
-            }}
+                loading="lazy"
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="col-3 col-medium-3">
+            <h2 class="p-heading--3">Digital signage</h2>
+            <p>A small footprint and full OpenGL make Ubuntu Core the perfect platform for secure digital signs. Try Ubuntu Frame, a reliable and secure display server for embedded Linux devices with long term support.</p>
+            <p><a href="/engage/Embedding_IoT_GUI_with_Ubuntu_Frame">Guide to developing GUIs for IoT&nbsp;&rsaquo;</a></p>
           </div>
-        </div>
-        <div class="col-3 col-medium-3">
-          <h2 class="p-heading--3">Automotive</h2>
-          <p>Reinvent the automobile with Ubuntu, designed for critical embedded systems.</p>
-          <p><a href="/automotive">Learn more&nbsp;&rsaquo;</a></p>
         </div>
       </div>
-    </div>
-    <div class="col-6 p-tile">
-      <div class="row p-tile__row">
-        <div class="col-3 col-medium-3 u-hide--small">
-          <div class="u-align--center">
-            {{
-              image(
-                url="https://assets.ubuntu.com/v1/51aec8b1-Networking+equipment.svg",
-                alt="Networking",
-                width="180",
-                height="165",
+      <div class="col-6 p-tile">
+        <div class="row p-tile__row">
+          <div class="col-3 col-medium-3 u-hide--small">
+            <div class="u-align--center">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/42bfe763-Automotive-car-chassi.svg",
+                alt="",
+                width="711",
+                height="480",
                 hi_def=True,
-              ) | safe
-            }}
+                loading="lazy"
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="col-3 col-medium-3">
+            <h2 class="p-heading--3">Automotive</h2>
+            <p>Trusted by major automotive companies, Canonical delivers a full suite of open source solutions and services to deliver security, safety and innovation for digital transformation in automotive.</p>
+            <p><a href="/engage/software-defined-vehicles-whitepaper">A CTO's guide to software-defined vehicles&nbsp;&rsaquo;</a></p>
           </div>
-        </div>
-        <div class="col-3 col-medium-3">
-          <h2 class="p-heading--3">Networking</h2>
-          <p>Build secure SmartNICs and operate them at scale on Ubuntu.</p>
-          <p><a href="/internet-of-things/networking">Learn more&nbsp;&rsaquo;</a></p>
         </div>
       </div>
     </div>
-    <div class="col-6 p-tile">
-      <div class="row p-tile__row">
-        <div class="col-3 col-medium-3 u-hide--small">
-          <div class="u-align--center">
-            {{
-              image (
+  </div>
+  <div class="p-section u-no-padding--bottom">
```

Comment:
`p-section` only adds bottom padding so these can be omitted altogether to get the same spacing